pulpcore/cli/common/generic.py

```diff
@@ -1002,29 +1000,30 @@ def version_callback(
         )
         return repo_ver_ctx
 
-    @click.command("list")
-    @click.option("-t", "--type", "type", type=click.Choice(names), default=names[0])
+    @pulp_command("list")
+    @click.option("--all-types", is_flag=True)
```

Comment:
I've changed the default behavior of this command group. It might make more sense to pull this out into a separate PR.

Reply:
What motivates the change?